aggregator.py

```diff
@@ -605,6 +617,10 @@ def __init__(self, hostname, interval=1.0, expiry_seconds=300, formatter=None, r
             'ms': Histogram,
             's': Set,
         }
+        self.histogram_aggregates = histogram_aggregates\
```

Comment:
That should probably be in the Aggregator class so you don't have to duplicate it in all its subclasses.